src/retrace/retrace.py

```diff
@@ -1839,7 +1839,14 @@ def prepare_debuginfo(self, vmcore, chroot=None, kernelver=None, crash_cmd=["cra
         else:
             child = Popen(crash_cmd + ["-s", vmcore, vmlinux], stdin=PIPE, stdout=PIPE, stderr=STDOUT,
                           encoding='utf-8')
-        stdout = child.communicate("mod\nquit")[0]
+        try:
+            t = 3600
+            if CONFIG["ProcessCommunicateTimeout"]:
+                t = CONFIG["ProcessCommunicateTimeout"]
+            stdout = child.communicate("mod\nquit", timeout=t)[0]
+        except:
```

Comment:
Can you use a specific exception (`TimeoutExpired`) here?

Reply:
Sure can!